code/unclassified/src/tokenizer/tokenizer.cpp

```diff
@@ -2,6 +2,8 @@
 #include <cstring>
 // Part of Cosmos by OpenGenus Foundation
 
+using namespace std;
+
 char *myStrTok(char *input, char delim)
```

Comment:
If you plan on fixing this file, I would figure out a way to replace all usage of `char*` with `std::string`, since it's the best C++ practice.

Reply:
I must still use a pointer. Instead of ```char*``` I must use ```string*```. Because the function tidy up after its termination.

Reply:
@christianbender The point is, with `std::string`, you won't have to do tidying up at all! It might be a bit of a drastic change, but it is beneficial.